packages/app/src/cli/services/app-logs/poll-app-logs.ts

```diff
@@ -68,9 +66,9 @@ export const pollAppLogs = async ({
           }).catch((error) => {
             outputDebug(`Unexpected error during polling: ${error}}\n`)
           })
-        }, POLLING_BACKOFF_INTERVAL_MS)
+        }, POLLING_THROTTLE_RETRY_INTERVAL_MS)
       } else {
-        throw new Error(`Error while fetching: ${responseText}`)
+        throw new Error(`Unhandled bad response: ${response.status}`)
```

Comment:
Maybe we should throw a specific error and catch that below to reset the timeout. Then we can catch any other random errors and handle those separately - maybe kill the polling for any _unexpected_ errors for example.

Reply:
This code branch here is specifically for bad responses from the server.
When this throws, it does end up being caught by our overarching try/catch.

We had the distinction before of only retrying on these "bad response from server" errors, and stopping the polling on anything unexpected.
But sometimes the CLI would error out on other transient things that weren't covered here, ie. `ECONNRESET`, so we decided to retry on unexpected things as well.

This throw, coupled with the overarching try/catch, means that our catch can handle the retries in both scenarios (bad response from server, and any other unknown failure)

Reply:
Got it. Thanks